keyboards/handwired/wwa/soyuz/keymaps/default/rules.mk

```
@@ -0,0 +1 @@
+ENCODER_MAP_ENABLE = yes
```

This still needs the newline as previously requested.
```suggestion
ENCODER_MAP_ENABLE = yes

```